docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/github/index.md

```diff
@@ -0,0 +1,182 @@
+---
+sidebar_label: GitHub
+---
+
+# The GitHub Webhook Receiver
+
+The Github Webhook Receiver will respond to ping, push, and package events.
+For ping events, a 200 is returned to verify connectivity. Push events refresh
+any Warehouses subscribed to the repository from which the event originated. A
+package event refreshes any Warehouses subscribed to images from a ghcr.io
+package registry.
+
+The following instructions will work for Github,
+Github Enterprise Cloud, and GitHub Enterprise Server.
+
+## Configuring the Receiver
+
+The GitHub webhook receiver will need to reference a Kubernetes `Secret` with a
+`secret` key in its data map. This
+[shared secret](https://en.wikipedia.org/wiki/Shared_secret) will be used by
+GitHub to sign requests. The receiver will use it to authenticate those requests
+by verifying their signatures.
+
+:::note
+The following command can be used to generate a sufficiently secure secret:
+
+```shell
+openssl rand -base64 48 | tr -d '=+/' | head -c 32
+```
+
+:::
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gh-wh-secret
+  namespace: kargo-demo
+stringData:
+  secret: <your-secret-here>
+---
+apiVersion: kargo.akuity.io/v1alpha1
+kind: ProjectConfig
+metadata:
+  name: kargo-demo
+  namespace: kargo-demo
+spec:
+  webhookReceivers: 
+    - name: gh-wh-receiver
+      github:
+        secretRef:
+          name: gh-wh-secret
+```
+
+## Retrieving the Receiver's URL
+
+Kargo will generate a hard-to-guess URL from the configuration. We can obtain 
+this URL using the following command:
+
+```shell
+  kubectl \
+    get projectconfigs \
+    kargo-demo \
+    -n kargo-demo \
+    -o=jsonpath='{.status.webhookReceivers}'
+```
+
+
+## Registering with Github
+
+When configuring on Github, you can configure either a webhook or an app. We will outline instructions for both, starting with webhooks.
+
+### Webhooks
+
+1. Navigating to the Webhooks Dashboard
+
+    1. Navigate to <Hlt>Settings</Hlt>
+    1. Click <Hlt>Webhooks</Hlt>
+    1. Click <Hlt>Add webhook</Hlt>
```

Comment:
You're still not mentioning the starting point for this. You can address that and simultaneously reduce the number of steps with something like this:

```
1. Navigate to `https://github.com/<account>/<repository>/settings/hooks` where
   `<account>` has been replaced with your GitHub username or an organization
   for which you are an administrator and `repository` has been replaced with
   the name of a repository belonging to that account.
```